.golangci.yml

```diff
@@ -28,7 +27,6 @@ linters:
     - typecheck
     - unconvert
     - unparam
-    - unused
```

Comment:
Why are we removing linting checks? That seems like a bad strategy.

Reply:
Added them back and addressed linting checks: https://github.com/celestiaorg/iavl/pull/5/commits/e77541ebe681486143c3f313ac6b7b1b8e2ef5e5